modules/library/xml/src/main/java/org/geotools/xml/styling/SLDParser.java

```diff
@@ -973,6 +973,8 @@ protected Rule parseRule(Node ruleNode) {
                 symbolizers.add(parseTextSymbolizer(child));
             } else if (childName.equalsIgnoreCase("RasterSymbolizer")) {
                 symbolizers.add(parseRasterSymbolizer(child));
+            } else if (childName.equalsIgnoreCase(VendorOptionString)) {
```

Comment:
Nice. But I don't see the encoding in SLDTransformer, should be added too (the SLD XSD bindings instead are read only, no worries there)

Reply:
Actually wait, the support for vendor options should be added also in RuleBuilder, in gt-brewer

Reply:
fixed